test/integration/simple_example/controls/gcloud.rb

```diff
@@ -68,4 +68,13 @@
     its(:stdout) { should match "projects/#{attribute("project")}/topics/example-topic\n" }
   end
 
+  describe command("gcloud --project=#{attribute("project")} beta healthcare hl7v2-stores describe example-hl7v2 --dataset=example-healthcare-dataset") do
+    its(:exit_status) { should eq 0 }
+    its(:stderr) { should eq "" }
+    its(:stdout) {
+      should include "parserConfig"
+      should include "SOFT_FAIL"
```

Comment:
You probably need two its(:stdout) {} blocks for the two matches. From the test result, it only shows 

```
     ✔  Command: `gcloud --project=ci-healthcare-9b8a beta healthcare hl7v2-stores describe example-hl7v2 --dataset=example-healthcare-dataset` stdout should include "SOFT_FAIL"
```

Reply:
Done. FWIW it definitely performs the assertions in the block (I know because a previous version failed the first one).  Updating for better logs.

Reply:
Thanks! Good to know!